### DISCUSSION TITLE
Second option for Retraction - Lenght for a long move

### DISCUSSION THREAD

**Discussion by PitFx** [OrcaSlicer/OrcaSlicer#3244] [Ideas]
Hello, I have an idea for a second option for Retraction.
- Lenght for a long move
- - Travel distance threshold for long move

This setting would be used for long moves according to the set threshold.

And why? Because it could help with filament leakage on long moves.

Thank you, Peter.

**Comment by LazaroFilm** [comment]
This would be a great idea. When I use PETG with fast speeds I need to bring the temperature up quite a bit which makes the filament ooze more than normal. Retracting more on long moves would solve that problem without adding too much time retracting too much systematically on every move.
